python/client/clients/cli.py

```diff
@@ -0,0 +1,136 @@
+
+
+from GrpcClient import GrpcClient
```

Comment:
Please follow my changes in https://github.com/catmajor/ARFlow/pull/1/files to update this and other files as well.